features/guidance/advanced-lanes.feature

```diff
@@ -41,8 +44,10 @@ Feature: Turn Lane Guidance
         Given the node map
             """
                           e
-            a     b       c g
-                          d
+            a . . b . . . c g
+                    `     .
+                       `  .
```

Comment:
even more so here. Please check all test modifications whether they are intended or not